Maple2.Database/Storage/Game/GameStorage.Club.cs

```diff
@@ -43,17 +44,21 @@ public partial class Request {
         }
 
         public IList<ClubMember> GetClubMembers(long clubId) {
-            return (from member in Context.ClubMember where member.ClubId == clubId
-                    join account in Context.Account on member.Character.AccountId equals account.Id
-                    join indoor in Context.UgcMap on
-                        new {OwnerId=member.Character.AccountId, Indoor=true} equals new {indoor.OwnerId, indoor.Indoor}
-                    join outdoor in Context.UgcMap on
-                        new {OwnerId=member.Character.AccountId, Indoor=true} equals new {outdoor.OwnerId, outdoor.Indoor} into plot
-                    from outdoor in plot.DefaultIfEmpty()
-                    select new ClubMember(
-                        BuildPlayerInfo(member.Character, indoor, outdoor, account.Trophy),
-                        member.CreationTime.ToEpochSeconds(),
-                        member.Character.LastModified.ToEpochSeconds())).ToList();
+            var result = (from member in Context.ClubMember where member.ClubId == clubId
+                          join account in Context.Account on member.Character.AccountId equals account.Id
+                          join indoor in Context.UgcMap on
+                              new {OwnerId=member.Character.AccountId, Indoor=true} equals new {indoor.OwnerId, indoor.Indoor}
+                          join outdoor in Context.UgcMap on
+                              new {OwnerId=member.Character.AccountId, Indoor=true} equals new {outdoor.OwnerId, outdoor.Indoor} into plot
+                          from outdoor in plot.DefaultIfEmpty()
+                          select new {account, member, indoor, outdoor})
+                .ToList();
```

Comment:
curious whether or not we even need `ToList()` here since we are gonna enumerate it again.

Reply:
It looks like it won't take it without it being a list.

Reply:
what about `AsEnumerable` which is a soft cast from `IQueryable` rather than needing to allocate a list.